docs/docs/dev/documentation/dps-and-data-model.md

```diff
@@ -104,19 +104,21 @@ As mentioned before, this is only required in Universal. In Kubernetes no change
 
 `kuma-dp` is built on top of `Envoy`, which has a powerful [Admin API](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) that enables monitoring and troubleshooting of a running dataplane.
 
-By default, `kuma-dp` starts `Envoy Admin API` on the loopback interface (that is only accessible from the local host) and the first available port from the range `30001-65535`.
+By default, `kuma-dp` starts `Envoy Admin API` on the loopback interface (that is only accessible from the local host) 
+and port is taken from the data plane resource field `networking.admin.port`. If the `admin` section is empty or port
+is equal to zero then the default value for port will be taken from the Kuma Control Plane configuration:
 
-If you need to override that behaviour, you can use `--admin-port` command-line option or `KUMA_DATAPLANE_ADMIN_PORT` environment variable.
-
-E.g.,
-
-* you can change the default port range by using `--admin-port=10000-20000`
-* you can narrow it down to a single port by using `--admin-port=9901`
-* you can turn `Envoy Admin API` off by using `--admin-port=`
+```yaml
+# Configuration of Bootstrap Server, which provides bootstrap config to Dataplanes
+bootstrapServer:
+  # Parameters of bootstrap configuration
+  params:
+    # Port of Envoy Admin
+    adminPort: 9901 # ENV: KUMA_BOOTSTRAP_SERVER_PARAMS_ADMIN_PORT
+```
 
-::: warning
-If you choose to turn `Envoy Admin API` off, you will not be able to leverage some of `Kuma` features, such as enabling `Prometheus` metrics on that dataplane.
-:::
+On Kubernetes there is no way for user to modify data plane proxy resource, that's why overriding of admin port value
+could be done using Pod's annotation `kuma.io/envoy-admin-port`.
```

Comment:
It is not possible to override the data plane proxy resource directly in Kubernetes.
If you still want to override it, use the pod annotation `kuma.io/envoy-admin-port`.